### DISCUSSION TITLE
Is there a way to just disable traces?

### DISCUSSION THREAD

**Discussion by fede843** [open-telemetry/opentelemetry-java-instrumentation#10374] [Q&A] [answered]
Hello, I am using the agent to fetch metrics and traces in most of our microservices. But in some few, I am exporting only metrics. There is no need for traces there. The problem is that those services are talking to some others with traces on. I am loosing the root parent trace because of this.
What I would like to do, is to keep the same configuration, and disable the tracing mechanism in the unneeded services, used only for metrics.
I am just not exporting them, but that does not prevent for traces to be created and sent out in headers.

**Comment by laurit** [comment, marked as answer]
Have you tried setting [`otel.propagators`](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#propagator) to `none`? This should prevent trace context from propagating to other services.

**Comment by fede843** [comment]
that was genius, I was not aware of that one. Thanks a lot!
Should "none" be listed over there? Is it implicit somehow?

**Reply by trask** [reply]
> Should "none" be listed over there?

oh, yes, I believe so, can you open an issue or send a PR for that? (for reference, it's listed in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration)
